system-tests/project-fixtures/react/vite.config.js

```diff
@@ -1,5 +1,11 @@
 const {defineConfig} = require('vite')
 
 module.exports = defineConfig({
+  resolve: {
```

Comment:
I had to add aliases for these packages, otherwise the React 17 tests that use Vite would fail when I called `React.useState` in my test.